python/python/lance/__init__.py

```diff
@@ -77,6 +78,10 @@ def dataset(
         If specified, use the provided commit handler to lock the table while
         committing a new version. Not necessary on object stores other than S3
         or when there are no concurrent writers.
+    index_cache_size : optional, int
+        Index cache size. Index cache is a LRU cache with TTL. This number specify the
+        number of index page, for example, an IVF partition, to be cached in
```

Comment:
This is for indexing OOM? Could you provide an example for users to estimate the max amount of memory used for the cache given parameter values? As it stands it's unclear how to translate from index pages to GBs

Reply:
Added more explains.